queries.yaml

```diff
@@ -975,19 +958,77 @@ post_process:
   - fn: vectordatasource.transform.drop_properties_with_prefix
     params: {prefix: mz_}
 
+  # drop small inners in buildings before we do the merge.
+  - fn: vectordatasource.transform.drop_small_inners
+    params:
+      end_zoom: 15
+      source_layers: [buildings]
+      pixel_area: 0.25
+
   - fn: vectordatasource.transform.merge_building_features
     params:
       source_layer: buildings
       start_zoom: 13
-      end_zoom: 15
+      end_zoom: 16
       quantize:
-        13: vectordatasource.transform.quantize_height_round_nearest_10_meters
-        14: vectordatasource.transform.quantize_height_round_nearest_5_meters
+        13: vectordatasource.transform.quantize_height_round_nearest_20_meters
+        14: vectordatasource.transform.quantize_height_round_nearest_10_meters
+        15: vectordatasource.transform.quantize_height_round_nearest_10_meters
+      # todo: keep instead of drop?
       drop:
         - name
+        - short_name
         - addr_housenumber
         - addr_street
-      max_merged_features: 1000
+        - osm_relation
+      # NOTE: max_merged_features is set to keep the time taken for geometry
+      # merging down, as it seems to go up with the square of the number of
+      # features.
+      max_merged_features: 2000
```

Comment:
Wouldn't this mean that the size of the tile would stay larger than we'd like though? And would we want to try another strategy in these cases, like drop some buildings from even appearing that don't rank high enough (not sure this is possible), or increase the buffer size we are using? Maybe it's smarter to leave it and just track this somehow so after runs we can see which tiles triggered the max?